src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,25 @@
-export const PostInfo = () => <>Put the post here</>;
+import { UserInfo } from '../UserInfo/index';
+import { CommentList } from '../CommentList/index';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+
+      <p>
+        {' Posted by  '}
+
+        <UserInfo user={post.user} key={post.userId} />
```

Comment:
```suggestion
        {post.user && <UserInfo user={post.user}  />}
```
why do you need key prop here?
post.user can be undefined, so better to render UserInfo component conditionally